src/components/audio-params.js

```diff
@@ -173,39 +177,44 @@ AFRAME.registerComponent("audio-params", {
 
```

Comment:
This component is still attempting to write directly to `this.data` in its `init` method. The contract for setting component `data` is calling `setAttribute`. Basically, `this.data` is meant to be read-only.

Reply:
Updated